### ISSUE TITLE
CASE, empty nested block not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#3011]
https://rules.abaplint.org/empty_structure/

```abap
DATA bar TYPE i.
CASE bar.
  WHEN 1.
    CASE bar.
      WHEN '00' OR '10'.
      WHEN '01'.
      WHEN '11'.
      WHEN OTHERS.
    ENDCASE.
  WHEN OTHERS.
    ASSERT 1 = 'todo'.
ENDCASE.
```
